spdx-utils/src/main/kotlin/SpdxLicenseMapping.kt

```diff
@@ -19,337 +19,332 @@
+            "CPL" to CPL_1_0.toExpression(),
+            "Cecill-C" to CECILL_C.toExpression(),
+            "Common Development and Distribution License" to CDDL_1_0.toExpression(),
+            "Common Development and Distribution License (CDDL) v1.0" to CDDL_1_0.toExpression(),
+            "Common Development and Distribution License (CDDL), Version 1.1" to CDDL_1_1.toExpression(),
+            "Common Public License - v 1.0" to CPL_1_0.toExpression(),
+            "Common Public License Version 1.0" to CPL_1_0.toExpression(),
+            "Creative Commons 3.0 BY-SA" to CC_BY_SA_3_0.toExpression(),
+            "Dual License: CDDL 1.0 and GPL V2 with Classpath Exception" to (CDDL_1_0 and GPL_2_0),
+            "Dual license consisting of the CDDL v1.1 and GPL v2" to (CDDL_1_1 and GPL_2_0),
+            "EPL" to EPL_1_0.toExpression(),
+            "EPL (Eclipse Public License), V1.0 or later" to EPL_1_0.toExpression(),
+            "Eclipse Distribution License (EDL), Version 1.0" to BSD_3_CLAUSE.toExpression(),
+            "Eclipse Distribution License (New BSD License)" to BSD_3_CLAUSE.toExpression(),
+            "Eclipse Distribution License - v 1.0" to BSD_3_CLAUSE.toExpression(),
+            "Eclipse Distribution License v. 1.0" to BSD_3_CLAUSE.toExpression(),
+            "Eclipse Public License" to EPL_1_0.toExpression(),
+            "Eclipse Public License (EPL)" to EPL_1_0.toExpression(),
+            "Eclipse Public License (EPL) 1.0" to EPL_1_0.toExpression(),
+            "Eclipse Public License (EPL), Version 1.0" to EPL_1_0.toExpression(),
+            "Eclipse Public License - Version 1.0" to EPL_1_0.toExpression(),
+            "Eclipse Public License - v 1.0" to EPL_1_0.toExpression(),
+            "Eclipse Public License 1.0" to EPL_1_0.toExpression(),
+            "Eclipse Public License v1.0" to EPL_1_0.toExpression(),
+            "Eclipse Public License v2.0" to EPL_2_0.toExpression(),
+            "Eclipse Public License, Version 1.0" to EPL_1_0.toExpression(),
+            "Eclipse Publish License, Version 1.0" to EPL_1_0.toExpression(),
+            "FreeBSD" to BSD_2_CLAUSE_FREEBSD.toExpression(),
+            "GNU Affero General Public License v3" to AGPL_3_0.toExpression(),
+            "GNU GENERAL PUBLIC LICENSE Version 2, June 1991" to GPL_2_0.toExpression(),
+            "GNU GPL v2" to GPL_2_0.toExpression(),
+            "GNU General Lesser Public License (LGPL) version 2.1" to LGPL_2_1.toExpression(),
+            "GNU General Public Library" to GPL_3_0.toExpression(),
+            "GNU General Public License (GPL)" to GPL_3_0.toExpression(),
+            "GNU General Public License (GPL), version 2, with the Classpath exception" to GPL_2_0.toExpression(),
+            "GNU General Public License Version 2" to GPL_2_0.toExpression(),
+            "GNU General Public License v2 or later (GPLv2+)" to GPL_2_0PLUS.toExpression(),
```

Comment:
`GPL_2_0PLUS` (and friends) is also a deprecated license. We should use `GPL_2_0_OR_LATER` instead.

Reply:
I have added a separate commit to remove all deprecated licenses from the mappings and added a test for it.